src/analytics/patch/tests.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { MockedProvider } from "@apollo/react-testing";
+import { render, fireEvent, queryHelpers } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { GET_USER, GET_PATCH, GET_PATCH_TASK_STATUSES } from "gql/queries";
+import { PatchTabs } from "pages/patch/PatchTabs";
+import { ContextProviders } from "context/Providers";
+import { GET_PATCH_EVENT_DATA } from "analytics/patch/query";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+/**
+ * UNABLE TO GET THESE TESTS TO WORK AFTER 5 HOURS OF EFFORT
+ * MADE A TICKET TO REVISIT INTEGRATION TESTS AT A LATER DATE
```

Comment:
Can you please link the TicketNumber in the comment for reference

Reply:
done